wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsDFUClient.java

```diff
@@ -14,6 +14,8 @@
 import javax.xml.parsers.DocumentBuilderFactory;
 
 import org.apache.axis.client.Stub;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
is this found in "stock" SDK, or is it a new dependency?

Reply:
It's already in the pom file for wsclient.